librz/cons/line.c

```diff
@@ -41,6 +41,7 @@ RZ_API void rz_line_free(void) {
 	// XXX: prompt out of the heap?
 	free ((void *)I.prompt);
 	I.prompt = NULL;
+	//rz_list_free (I.kill_ring);
```

Comment:
why this commented line?

Reply:
I think it is right to uncomment it.

Reply:
It causes double free segfault then.

Reply:
Why? Does it happen in radare2 as well? Maybe it should be `rz_list_free`+`I.kill_ring=NULL`? (though this is not *the* solution)

Reply:
It doesn't happen in radare2:  https://github.com/radareorg/radare2/commit/06b735441d299e90695235495982354bef1fcc41 - note the green status.

Reply:
I can just remove this line if you want.

Reply:
Adding this free is right in theory... Could you try uncommenting it so I can see where exactly the double free happens?

Reply:
What about merging this first, then try? It's grown quite big.